frontend/src/constants.js

```diff
@@ -2,7 +2,7 @@ export const PORT = process.env.PORT || 2856;
 export const API_BASE_URL =
   process.env.REACT_APP_API_PROD_BASE_URL || `http://localhost:${PORT}`;
 export const CLIENT_BASE_URL =
-  process.env.REACT_APP_CLIENT_PROD_BASE_URL || `http://localhost:3000`;
+  process.env.REACT_APP_CLIENT_PROD_BASE_URL || "http://localhost:3000";
```

Comment:
The "3000" is definitely a mistake and should be replaced by `PORT` variable on line 1 in string templates. Please address it since it's a small thing. Unless you're willing to open a separate issue for it... your choice.

Reply:
Sure thing 👍🏾